api_specifications/order-api-0.1.0.yaml

```diff
@@ -235,7 +235,7 @@ components:
     Order:
       description: An order is created as an outcome of an hearing or based on an application. Order will contain a set of tasks
       type: object
-      required: [ tenantId, createdDate, orderType, status, isActive ]
+      required: [ tenantId, createdDate, orderType, isActive ]
```

Comment:
Is createdDate in epoch time? Let's add that in the description.

Reply:
yes. added description and updated type of field